packages/components/tools/templates.js

```diff
@@ -84,6 +84,22 @@ const loadContents = glob =>
   getContents(glob).then(contents => {
     contents.forEach((content, templateName) => {
       Handlebars.registerPartial(templateName, content);
+      Handlebars.registerHelper('isFirstDataTd', function(columns, index) {
+        var firstDataTdIndex = 0;
+        for (var i = 0; i < columns.length; ++i) {
+          if (
+            columns[i].section |
+            columns[i].checkbox |
+            columns[i].additionalIcons |
+            columns[i].menu
+          ) {
+            ++firstDataTdIndex;
```

Comment:
Did you want to bail out the loop here given you seems to be looking for the first column meeting the condition?

Reply:
Did you mean `return index === columns.findIndex(column => !column.checkbox && !column.additionalIcons && !column.menu)`?

Reply:
yes, that's definitely a more succinct way to put it :)
fixed in https://github.com/carbon-design-system/carbon/pull/2992/commits/104c1b9cafbffe7cdd4b12f20e3b5b4aadce40c7